core/middleware/region.py

```diff
@@ -11,10 +13,13 @@ def middleware(request):
         ):
             return get_response(request)
 
+        admin_request = bool(re.match(r"^/[a-z]{2}/admin", request.path))
```

Comment:
What's wrong with the  `request.path.startswith(reverse("admin:index"))`?

Reply:
I would need to list all possible locales since when we enable translations format is `/en/admin`